examples/todo-list/flutter-todo-list/lib/functions/auth.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:supabase_flutter/supabase_flutter.dart';
+
+class AuthSupabase {
+  static final client = SupabaseClient(
+      "https://newknrbzotfpvatooqxz.supabase.co",
+      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Im5ld2tucmJ6b3RmcHZhdG9vcXh6Iiwicm9sZSI6ImFub24iLCJpYXQiOjE2NTQwMjM2ODYsImV4cCI6MTk2OTU5OTY4Nn0.KI6TzzzailhNRfRAeJMIDtMVZSBZbaliDK2K_J0VVn8");
```

Comment:
We should avoid creating multiple Supabase clients in a single app. We can always use `Supabase.instance` wherever we want to access Supabase within the app. We can probably get rid of this one.

Reply:
DONE ✅